modules/issue_tracker/php/issuerowprovisioner.class.inc

```diff
@@ -96,8 +96,9 @@ class IssueRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisioner
             $module = &$modules[$mname];
         }
 
-        $cid = $row['centerId'];
+        $cids = isset($row['centerId']) ?
+            [$row['centerId']] : array_keys(\Utility::getSiteList(false));
```

Comment:
Can the getSiteList part of this be brought up to the constructor so that it's cached between rows? We shouldn't be re-querying the database to get the site list for every row of the table, that won't scale.

Reply:
Sure! So move it up to the constructor and set it as a class variable?

Reply:
Yeah, that way we query the database once per page load instead of once per row.